string robot_pose_name
---
RobotPose robot_pose
bool success
